src/components/textarea/textarea.spec.js

```diff
@@ -413,6 +416,74 @@ describe("componentWillUnmount", () => {
       expect(label.prop("isRequired")).toBe(true);
     });
   });
+
+  describe("new validations", () => {
+    const renderWithNewValidations = ({ error, warning }) =>
+      mount(
+        <CarbonProvider validationRedesignOptIn>
+          <Textarea
+            labelHelp="Example hint text"
+            error={error}
+            warning={warning}
+            labelAlign="left"
+            labelInline
+            labelWidth={100}
+            reverse
+          />
+        </CarbonProvider>
+      );
+
+    describe("label width and align props", () => {
+      it("default to undefined", () => {
+        const wrapper = renderWithNewValidations({});
+        const { labelAlign, labelInline, reverse, labelWidth } = wrapper
+          .find(FormField)
+          .props();
+
+        expect(labelAlign).toEqual(undefined);
+        expect(labelInline).toEqual(undefined);
+        expect(reverse).toEqual(undefined);
+        expect(labelWidth).toEqual(undefined);
+      });
+    });
+
+    describe("hint/ labelHelp", () => {
+      it("is visible when the prop is passed", () => {
+        const wrapper = renderWithNewValidations({});
+        expect(wrapper.find(StyledHintText).text()).toEqual(
+          "Example hint text"
+        );
+      });
+
+      it("applies the expected styling", () => {
+        const wrapper = renderWithNewValidations({});
+
+        assertStyleMatch(
+          {
+            fontSize: "14px",
+            marginTop: "0px",
+            marginBottom: "8px",
+            color: "var(--colorsUtilityYin055)",
+          },
+          wrapper.find(StyledHintText)
+        );
+      });
+    });
+
+    describe("new validation design", () => {
+      it("error message is visible when the prop is passed", () => {
```

Comment:
Would it be worth tweaking these two tests slightly - such that it checks the message and type (error or warning) is correct too? In my opinion these could be more explicit :)

Reply:
I don't think so no as Chromatic will capture the difference between error and warning for us, good idea though 👍🏻